ci_build/download_ort_release.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e -x
+
+ORT_VER="1.13.1"
+
+while getopts i: parameter_Option
+do case "${parameter_Option}"
+in
+i) ORT_VER=${OPTARG};;
+esac
+done
+
+apt-get update
+apt-get install -y cmake gcc g++ libpng-dev libjpeg-turbo8-dev curl
```

Comment:
nit: this is not really part of "downloading ORT", maybe it could go in a separate script or this one could be renamed

Reply:
Updated.